modules/ROOT/pages/runtime-configure-gov.adoc

```diff
@@ -0,0 +1,83 @@
+= To Configure the Mule Runtime to Connect to the Government Cloud
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To use Mule Runtime within the Government Cloud, you must redefine the configuration properties that enable Mule Runtime to point to the EU control plane. You must also provide the Client ID and Client Secret of your environment.
+
+By default, the Mule Runtime is configured to point to the US control plane. If you do not override these properties, the Mule Runtime modules are disabled in the EU control plane.
+
+The properties you must override depend on the version of the Mule runtime you are using, as described in the following sections.
+
+== Configuring Mule Runtime 4.0
+
+=== wrapper.conf
+
+To configure Mule Runtime using the `wrapper.conf` file, modify the following properties:
+
+----
+wrapper.java.additional.<n>=-Danypoint.platform.client_id=<environment client ID>
```

Comment:
It can be environment or organization credentials.

Reply:
I've added some extra text. Please review to make sure I did not misunderstand your meaning. Thank you!

Reply:
Credentials (ie client ID and client secret) may come in the case of Mule 4 from a specific Environment (eg Production, Sandbox, etc), the organization or business groups.

Structure example:

**Master organization**
- Environment A
- Environment B
- **Business Group A** (aka internally as sub organizations)
    - Environment A
    - Environment H
    - Environment I
- **Business Group B** (aka internally as sub organizations)
    - Environment H

Reply:
-> So in fact you may choose either organization or environment credentials (recommended way the second one).

Reply:
OK - thank you for the clarification. I have updated the occurrences back to 'environment'.